src/Illuminate/Foundation/Console/Kernel.php

```diff
@@ -113,6 +114,12 @@ public function handle($input, $output = null)
 
             $this->renderException($output, $e);
 
+            if (! $input->hasParameterOption(['--help', '-h'], true) && $input->hasArgument('command')) {
```

Comment:
We should probably only look for `--help` here, since we're then calling `--help`, not `-h`.